luaui/Widgets/dbg_widget_auto_reloader.lua

```diff
@@ -0,0 +1,65 @@
+function widget:GetInfo()
+	return {
+		name = "Widget Auto Reloader",
+		desc = "Auto reloads all widgets that have changed when game is out of focus (and once on return focus)",
+		author = "Beherith, Floris",
+		date = "2024.03.12",
+		license = "GNU GPL v2",
+		layer = 0,
+		enabled = true, --  loaded by default?
```

Comment:
I also use editor with autosave and with this you dont even have to switch to the game to see if your edit fixes it. Does it really matter that it will error during the code editing? Maybe in really specific cases... but then you can also just disable the widget and toggle it